views/objecting-entity-name.html

```diff
@@ -60,11 +61,13 @@
           items: [
             {
               value: "yes",
-              text: "Yes"
+              text: "Yes",
+              checked: isYesChecked
             },
             {
               value: "no",
-              text: "No"
+              text: "No",
+              checked: isNoChecked
```

Comment:
Could this be simplified to one boolean?
isChecked, and this value be !isChecked

Reply:
I think these need to be dealt with separately so as to cover the 3 senarios:
Neither selected
Yes selected
No selected.

Reply:
I forgot about the Neither selected scenario, fair enough